config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "issuer": "https://example.oktapreview.com",
+  "audience": "ANRZhyDh8HBFN5abN6Rg",
+  "apiToken": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "authzIssuer": "https://example.oktapreview.com/oauth2/aus8q4gst8vbUGzFp0h7",
```

Comment:
Couldn't this be cleaned up by setting the `issuer` to your custom authorization server? I don't see why we need two issuers here.